PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/ArcadiaSparkBatchRunner.kt

```diff
@@ -66,8 +68,12 @@ class ArcadiaSparkBatchRunner : SparkBatchJobRunner() {
                         .first()
             } catch (ex: NoSuchElementException) {
                 throw ExecutionException(
-                        "Can't find Arcadia spark compute (${arcadiaModel.sparkWorkspace}:${arcadiaModel.sparkCompute})"
+                        "Can't find Arcadia Spark Compute (${arcadiaModel.sparkWorkspace}:${arcadiaModel.sparkCompute})"
                                 + " at tenant ${arcadiaModel.tenantId}.")
+            }  catch (ex: UninitializedPropertyAccessException) {
```

Comment:
We might have to check the if sparkCompute is initialized or not before, rather than catch the exception, just like NPE handling.